scripts/test.sh

```diff
@@ -25,4 +25,4 @@ echo "waiting on postgres to start..."
 
 echo "running tests..."
 
-NODE_ENV=test DB_CONNECTION_URI="postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST/postgres" yarn jest --runInBand $1
+NODE_ENV=test DB_CONNECTION_URI="postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST/postgres" yarn jest --runInBand "$@"
```

Comment:
This should _really_ be a separate commit.

Reply:
Yes apologies. This was done debugging and I forgot to split it out. It just passes all args passed to the script down to `jest` not just the first one.

Reply:
Can we pull this out into a seperate commit?